### DISCUSSION TITLE
Custom sort of milestone

### DISCUSSION THREAD

**Discussion by jpschewe** [community/community#11468] [Mobile]
When using GitHub via the web I can study the issues in a milestone be dragging then up and down the list. This sort is saved for the next time I view the milestone. If would be convenient if the mobile application allowed me to view the milestone in the same order.

**Comment by brianlovin** [comment]
Thanks for the feedback @jpschewe — I can see how not matching the sort order could be frustrating. We don't support the full Milestone view in the apps today, but I'll add your +1 to our tracking issues for this 😊
